src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31124,108 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033034 "An ON bipolar cell with a small dendritic tree that forms most of the central (invaginating) elements opposite the synaptic ribbon at the cone triad.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033034 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033034 "2023-04-04T09:51:44Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033034 "IMB cell")
+AnnotationAssertion(rdfs:label obo:CL_4033034 "invaginating midget bipolar cell")
+SubClassOf(obo:CL_4033034 obo:CL_0000749)
+
+# Class: obo:CL_4033035 (giant bipolar cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033035 "An ON bipolar cell that has large dendritic and axonal fields.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033035 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033035 "2023-04-04T09:54:15Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033035 "GB cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") rdfs:comment obo:CL_4033035 "A giant bipolar cell is usually monostratified, although, very rarely, it can be bistratified.")
+AnnotationAssertion(rdfs:label obo:CL_4033035 "giant bipolar cell")
+SubClassOf(obo:CL_4033035 obo:CL_0000749)
+
+# Class: obo:CL_4033036 (OFFx cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30712875") Annotation(oboInOwl:hasDbXref "PMID:31995762") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033036 "An OFF bipolar cell that is defined by the expression of NXPH1 and FEZF1.")
+AnnotationAssertion(obo:RO_0002175 obo:CL_4033036 obo:NCBITaxon_9443)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033036 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033036 "2023-04-04T10:54:15Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033036 "OFFx cell")
+SubClassOf(obo:CL_4033036 obo:CL_0000750)
+
+# Class: obo:CL_4033037 (diffuse bipolar 3 cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033037 "An OFF bipolar cell that has a large dendritic field and stratifies narrowly close to the middle of the Inner plexiform layer. Its axon terminal is characterized by regularly branching and varicose processes resembling beads on a string.")
```

Comment:
Inner --> inner

Reply:
After a cursory review, I found DB3a and DB3b mentioned in [PMID:32555229](https://pubmed.ncbi.nlm.nih.gov/32555229), but not this general class.

Reply:
In  [PMID:32032773](https://www.sciencedirect.com/science/article/pii/S1350946220300161?via%3Dihub):
[The DB3 cell identified by [Boycott and Wässle (1991)](https://www.sciencedirect.com/science/article/pii/S1350946220300161?via%3Dihub#bib18) from Golgi preparations has a comparatively large dendritic field and stratifies narrowly close to the middle of the IPL. Its axon terminal is characterized by regularly branching and varicose processes resembling beads on a string. Bipolar cells with this morphology were identified with antibodies against the [calcium binding protein](https://www.sciencedirect.com/topics/medicine-and-dentistry/calcium-binding-protein) [calbindin](https://www.sciencedirect.com/topics/medicine-and-dentistry/calbindin) in macaque, human and marmoset retinas]

Reply:
[PMID:32555229](https://pubmed.ncbi.nlm.nih.gov/32555229) is also listed as a dbxref. Did not find DB3 noted there.

Reply:
Yes, sorry, I am replying comments as I am doing the edits in protégé. That reference has been deleted once I push the commit.

Reply:
DB3 seems to be a related synonym of DB3a used before the discovery of DB3b. Therefore, I do not see a reason anymore to create this cell type, and DB3 should be a related synonym of DB3a with the appropriate references.